### DISCUSSION TITLE
Max Upload Size in Stream

### DISCUSSION THREAD

**Discussion by marcus-at-localhost** [atrocore/atrocore#609] [Q&A]
While upload works fine in general, the upload into the activity stream is not possible:

![image](https://user-images.githubusercontent.com/1337223/215067606-2fb7d161-6b5b-4f50-ae4e-b9e8ff51b86c.png)

In config.php I have:

```php
return [
   'attachmentUploadMaxSize' => 20000,
]
```

In my php.ini

```ini
upload_max_filesize=25M
post_max_size=20M
```

Any setting I missed?

Thanks!

Atrocore: v1.4.147

PS: you should have a button in Admin/Modules that copies all installed modules with version number as plain text to paste here. Probably helps to identify version related issues.

**Comment by rratsun** [comment]
We have created an internal issue for this. We will solve it soon.

**Comment by rzablodskyi** [comment]
Good afternoon. Regarding your problem, the `attachmentUploadMaxSize` parameter assumes that the value is in bytes. You can increase its value or remove this key from config.php, then the `post_max_size` value from php.ini will be used.

**Reply by marcus-at-localhost** [reply]
Thank you. Removing the setting solved it. The setting was probably a leftover from debugging the chunked upload many months ago. The DAM upload worked flawless though.
